helpers/ampersand/backstop.js

```diff
@@ -0,0 +1,38 @@
+import logger from 'gulp-logger'
+import colors from 'ansi-colors'
+import backstopjs from 'backstopjs'
+import { env, projectPath } from '../config'
+import configLoader from '../config-loader'
+
+export default callback => {
+  const backstopConfig = configLoader('backstop.json')
```

Comment:
Can we indent any custom stuff with 4 spaces?

Reply:
Turns out, you can do exactly that; I've had to update the .eslintrc file to add our overrides (and also to be JSON instead of YML as my editor was throwing a wobbly about it), but it works, and pretty well too!

Reply:
Turns out, you can do exactly that; I've had to update the .eslintrc file to add our overrides (and also to be JSON instead of YML as my editor was throwing a wobbly about it), but it works, and pretty well too!